src/components/shared/Layout/Layout.tsx

```diff
@@ -71,6 +74,7 @@ export function Layout() {
             flexDirection: "column",
             flexGrow: 1,
             overflowY: { xs: "unset", sm: "auto" },
+            mx: !isMobile ? 2 : undefined,
```

Comment:
I would revert this - it eats up another 32ish pixels of space on small-medium screens that are already struggling on space.
![Screen Shot 2024-10-26 at 14 44 32](https://github.com/user-attachments/assets/f3c5c82b-f59b-4459-9d08-ea99e3636de5)

Reply:
reverted